src/App.js

```diff
@@ -1,25 +1,45 @@
 import React from 'react';
 
+import { Clock } from './components/Clock';
 import './App.scss';
+import { Button } from './components/Button';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 'firstname',
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
-
-export default App;
+  render() {
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <p>
+          Current time:
+          {' '}
+          {this.state.isClockVisible && (<Clock name={this.state.clockName} />)}
+        </p>
+        <Button
+          actionName="Show Clock"
+          callback={() => {
```

Comment:
Краще уточнити, що це колбек, який буде трігеритися на onClick
```suggestion
          onClick={() => {
```